pages/common/minikube.md

```diff
@@ -0,0 +1,25 @@
+# minikube
+
+> Run Kubernetes on your local machine for free.
+
+- Start the cluster:
+
+`minikube start`
+
+- View the cluster details:
```

Comment:
Let's not put the kubectl commands here.
They belong to `kubectl`

Reply:
Hello, I got a few questions at this point, I will change and remove the `kubectl.  Thanks

Reply:
@mfrw I removed the references to `kubectl` and changed the description.